docs/cli-reference.md

```diff
@@ -244,7 +244,7 @@ rio export --riofile namespace/default
 
 ## image
 
-List images built from local registry
+List images built from a local registry
```

Comment:
Either use `the` or nothing, this means specifically the rio registry.

Reply:
Addressed via https://github.com/rancher/rio/pull/830/commits/441b69f05001c62f12d6a95ddeeed81e46b44a06